crates/test-utils/src/agent.rs

```diff
@@ -21,29 +21,17 @@ use hyperdrive_wrappers::wrappers::{
 use rand::{Rng, SeedableRng};
 use rand_chacha::ChaCha8Rng;
 use tokio::time::sleep;
-use tracing::{info, instrument};
+use tracing::instrument;
 
 use super::chain::ChainClient;
 
-#[derive(Copy, Clone, Debug)]
-enum Action {
-    Noop,
-    AddLiquidity(FixedPoint),
-    RemoveLiquidity(FixedPoint),
-    RedeemWithdrawalShares(FixedPoint),
-    OpenLong(FixedPoint),
-    CloseLong(FixedPoint, FixedPoint),
-    OpenShort(FixedPoint),
-    CloseShort(FixedPoint, FixedPoint),
-}
-
-#[derive(Clone, Default)]
+#[derive(Clone, Default, Getters)]
```

Comment:
What is `Getters` doing here? I see the crate let's you generate getters on private methods, but I'm not seeing the `#[getset...]` attribute added anywhere.

Reply:
Thanks, I added this and then removed it because it wasn't working as expected so I made the Wallet members public instead.